src/java.desktop/macosx/classes/sun/font/CCharToGlyphMapper.java

```diff
@@ -88,6 +88,31 @@ public synchronized boolean charsToGlyphsNS(int count,
         return false;
     }
 
+    private static int unicodeToUnits(int unicode, int dstOffset, char[] dst) {
+        if (unicode >= 0x10000) {
+            int base = unicode - 0x10000;
+            dst[dstOffset] = (char)((base >>> 10) + HI_SURROGATE_START);
+            dst[dstOffset+1] = (char)((base % 0x400) + LO_SURROGATE_START);
+            return 2;
+        } else {
+            dst[dstOffset] = (char) unicode;
+            return 1;
+        }
+    }
+
+    public int charToVariationGlyph(int unicode, int variationSelector) {
+        if (variationSelector == 0) return charToGlyph(unicode);
```

Comment:
We always use braces around the body, even for one line.

Reply:
Fixed